clouseau/Dockerfile

```diff
@@ -52,13 +52,41 @@ RUN git clone -b ntr_master https://github.com/neutrinity/clouseau . \
 # TODO tests need to get unskipped
 RUN mvn verify -Dmaven.test.skip=true
 
+# --- Stage3: Finalize Clouseau
+FROM erlang:18-slim as ntr-clouseau
+ENV CLOUSEAU_PATH /opt/clouseau
+ENV INDEX_DIR /index
+
+RUN apt-get -qq update -y \
+  && DEBIAN_FRONTEND=noninteractive apt-get -qq install -y apt-utils \
+  && DEBIAN_FRONTEND=noninteractive apt-get -qq install -y --no-install-recommends \
+  libnspr4 libnspr4-0d \
+  openssl \
+  curl \
+  ca-certificates \
+  unzip \
+  openjdk-7-jdk \
```

Comment:
I will replace this with JRE later (should get us another win on size), and also trim other deps from the final image (like, `ca-certificates` etc.).